ui/pages/destinations/[id]/index.js

```diff
@@ -1,971 +0,0 @@
-import { useEffect, useState, Fragment, useRef } from 'react'
```

Comment:
Do we want to keep this file for now, if we are "silently" shipping the new page?

Reply:
oh i think i did a bad rebase - so it came back! removing it!

Reply:
oh that is true lol oopsy!